sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -140,6 +140,8 @@ export interface QueueDescription {
    * for consumption by the next receiver. Settable only at queue creation time.
    * This is to be specified in ISO-8601 duration format
    * such as "PT1M" for 1 minute, "PT5S" for 5 seconds.
+   *
+   * ISO-8601 duration format: https://en.wikipedia.org/wiki/ISO_8601#Durations
```

Comment:
Perhaps add "More on"?

```suggestion
   * More on ISO-8601 duration format: https://en.wikipedia.org/wiki/ISO_8601#Durations
```

Reply:
Done